content/en/monitors/guide/as-count-in-monitor-evaluations.md

```diff
@@ -3,6 +3,7 @@ title: as_count() in Monitor Evaluations
 kind: guide
 aliases:
   - /monitors/guide/as-count-in-monitor-evaluations
```

Comment:
Remove this line since it is referencing itself

Reply:
done 👍